### PR TITLE
UX changes to pending payment page

### PR DESCRIPTION
**Pull request by pauljickling**
### What was wrong?

UX could be improved as described in issue #98

### How was it fixed?

- Added icons for QR Code, web3modal, and uniswap
- Added toggle button for QR Code (color based on bootstrap/Pretix config)
- Minor text improvements
- Changed links to buttons for Web3Modal and Uniswap (colors based on brand identity)
- Added some additional spacing between options
- Changed parentheticals to single unordered list items
- Generally tried to create a more consistent visual order
- Added a light gray background color to the address for manual payment option
- Made address text-wrap for smaller displays

Note that I did not attempt to change the currency display at the top per the reasoning from @davesque.

Screenshot:

pending

#### Cute Animal Picture

deer

### CODE REVIEW

File: pretix_eth/templates/pretix_eth/pending.html [hunk 1]
```diff
@@ -7,43 +7,44 @@
...
+    </li>
 
+    <li class="payment-options">
+      <img src="{% static "icons/uniswap-logo-32.svg" %}" />
+      <a target="blank" rel="noreferrer noopener" href="{{ uniswap_url }}">
+        <button class="btn btn-primary" id="uniswap_btn">{% trans "Uniswap" %}</button>
+      </a>
+      <ul class="payment-info">
+        <li>{% trans "Allows you to pay with any uniswap supported token." %}
+          <strong>{% trans "Please do not change the output amount!" %}</strong>
```
Comment (skylarweaver) [inline pretix_eth/templates/pretix_eth/pending.html:39]:
```suggestion
          <strong>{% trans "Please do not change the output amount or output currency!" %}</strong>
```

File: pretix_eth/templates/pretix_eth/pending.html [hunk 2]
```diff
@@ -7,43 +7,44 @@
...
-      </li>
-    </ol>
-  </p>
+    <li class="payment-options">
+      <img src="{% static "icons/web3modal-logo.png" %}" width=32 height=32 />
+      <a target="blank" rel="noreferrer noopener" href="{{ web3modal_url }}">
+        <button class="btn btn-primary" id="web3modal_btn">{% trans "Web3Modal" %}</button>
+      </a>
+      <ul class="payment-info">
+        <li>{% trans "Allows you to pay with WalletConnect and MetaMask" %}</li>
```
Comment (skylarweaver) [inline pretix_eth/templates/pretix_eth/pending.html:28]:
```suggestion
        <li>{% trans "Allows you to pay with WalletConnect or MetaMask." %}</li>
```

**Comment by pauljickling** [comment]
@skylarweaver your proposed changes sound good to me. What do you think @ligi ?

File: pretix_eth/templates/pretix_eth/pending.html (hunk 1 above)
Comment (ligi) [reply pretix_eth/templates/pretix_eth/pending.html:39]: :+1:

File: pretix_eth/templates/pretix_eth/pending.html [hunk 3]
```diff
@@ -7,43 +7,44 @@
...
-        ({% trans "allows you to pay with WalletConnect and MetaMask" %}).
-        <br/>
-      </li>
+  <p>{% trans "Choose one of the following payment options:" %}</p>
+  <ol>
+    <li class="payment-options">
+      <img src={% static "icons/qr_code.png" %} id="qr_icon" />
+      <button class="btn btn-primary" id="btn-qr">{% trans "Toggle QR Code" %}</button>
+      <ul class="payment-info">
+        <li>{% blocktrans trimmed %}Via <a href="https://eips.ethereum.org/EIPS/eip-681">ERC-681</a> QR-Code{% endblocktrans %}</li>
```
[the remark this answers is not in the document]
Comment (ligi) [reply pretix_eth/templates/pretix_eth/pending.html:16]: not sure if we need to mention "exact amount to the correct address." - sounds a bit weird to me

**Comment by ligi** [comment]
one suggestion (af840a9) I took - with the other one I am not yet sure and commented

**Comment by pauljickling** [comment]
Made some additional changes to the manual pay option. Included it on the top per Skylar's suggestion with the additional text provided below. I also added a copy to clipboard icon that the user can click so that it automatically copies the address to the clipboard. Haven't tested that on mobile browsers obviously, but it was working on Chromium, Firefox, and Safari desktop browsers. Of course users can always just select text and ctrl + c as normal, just a quality of life enhancement.

Other minor thing is I renamed the CSS file from pending.css to eth_plugin.css with the idea that if there are any other CSS changes that we want to make to the plugin we should just have a single file that manages that for all pages.

Demo (also includes the QR code toggle since I forgot to include that last time):

Screen Recording 2020-05-22 at 1 27 13 PM

**Comment by ligi** [comment]
looks nice - @kvbik can you deploy this one to staging?

**Pull request merged**